cabal.project

```diff
@@ -6,21 +6,25 @@ packages:
   cardano-crypto-class
   slotting
 
+-- Disable tests in a dependency (which are currently broken)
+package cardano-crypto
+  tests: False
```

Comment:
Just acknowledging that the problematic tests are being removed in https://github.com/input-output-hk/cardano-crypto/pull/65.

Reply:
By the time we update, the tests will be fixed in the PR Luke mentions. So lets not disable them pre-emptively.